services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -152,6 +174,12 @@
     @Field("waypoints") String waypointIndices,
     @Field("waypoint_names") String waypointNames,
     @Field("waypoint_targets") String waypointTargets,
-    @Field("enable_refresh") Boolean enableRefresh
+    @Field("enable_refresh") Boolean enableRefresh,
+    @Query("walking_speed") Double walkingSpeed,
```

Comment:
@Guardiola31337 My understanding is that `@Field` is only used for `POST` because it's `@FormUrlEncoded`, but if you think this shouldn't be the case let me know

Reply:
@Guardiola31337 nevermind I just understood what you were saying. It's interesting because I thought they were supposed to be `@Field` but the example test passed with `@Query`... I'll update them anyway